### PR TITLE
Add CORS details

### PR DESCRIPTION
**Pull request by brianorwhatever**
Text pulled from [here](https://w3c-ccg.github.io/did-method-web/#cross-origin-resource-sharing-cors-policy-considerations)

### CODE REVIEW

File: spec/security_and_privacy.md [hunk 1]
```diff
@@ -40,3 +40,9 @@ See also:
 
 - [UTS-46](https://unicode.org/reports/tr46/)
 - [[spec:RFC5895]]
+
+### Cross-Origin Resource Sharing (CORS) Policy Considerations
+
+To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] should be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response can be set to include the following header:
+
```
Comment (swcurran) [inline spec/security_and_privacy.md:47]: “can” — should that be a “SHOULD” in spec language, or in this part of the spec is that not appropriate?

Comment (brianorwhatever) [reply spec/security_and_privacy.md:47]: good call - maybe even a MUST?

**Comment by swcurran** [comment]
I think it is a **MUST**.  I think that is a common use case…

File: spec/security_and_privacy.md [hunk 2]
```diff
@@ -40,3 +40,9 @@ See also:
 
 - [UTS-46](https://unicode.org/reports/tr46/)
 - [[spec:RFC5895]]
+
+### Cross-Origin Resource Sharing (CORS) Policy Considerations
+
+To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] should be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response can be set to include the following header:
```
Comment (brianorwhatever) [inline spec/security_and_privacy.md:46]:
```suggestion
To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] needs to be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response MUST be set to include the following header:
```

**Comment by brianorwhatever** [comment]
added the MUST. this is ready now I think

File: spec/security_and_privacy.md [hunk 3]
```diff
@@ -40,3 +40,9 @@ See also:
 
 - [UTS-46](https://unicode.org/reports/tr46/)
 - [[spec:RFC5895]]
+
+### Cross-Origin Resource Sharing (CORS) Policy Considerations
+
+To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] needs to be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response MUST be set to include the following header:
```
Comment (andrewwhitehead) [inline spec/security_and_privacy.md:46]:
```suggestion
To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] needs to be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response MUST include the following header:
```

A bit less wordy

**Review by swcurran — APPROVED** [review]

**Pull request merged**